site/_posts/2020-05-04-setting-up-single-sign-on-for-nodebb.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: blog-post
+title: Setting up single sign on for nodebb 
+description: Using OIDC, we'll set up single sign on for a nodebb forum.
+author: Dan Moore
+image: blogs/dot-net-command-line-client/creating-user-cli-client.png
+category: blog
+excerpt_separator: "<!--more-->"
+---
+
+nodebb is a modern well supported forum software package. It has many plugins and provides a great user experience. Unlike hosted forums, you can run this on your own domain, which has UX and SEO benefits. 
+
+In this post, we'll show you how to set up single sign on (SSO) for your nodebb forum.
+
+<!--more-->
+
+## The dream
+
+Imagine you build out an application. Let's say you are building the world's best todo tracking app. Everyone loves it and it becomes wildly popular. Users are adding tasks and checking them off with abandon. Your customers clamor for a place to share all their task management tips. 
+
+Enter nodebb. You download and install it. The forum works great! Then you notice that every one of your users now must sign up for another username and password. Boo. 
+
+Luckily, you planned ahead and used an identity management system with OpenID Connect (OIDC) support. You realize you can set up nodebb to use this existing system to provide SSO to your users. All your todo app users will be able to log in and post in your forum with the same username and password they use for your app. Ah, the glory of single sign on. 
```

Comment:
I see sign-on hyphenated by most people and companies in the auth space. You're call but I vote for hyphenating it.
```suggestion
Ah, the glory of single sign-on. 
```